Topic1/src/com/topic1/MailList.java

```diff
@@ -0,0 +1,38 @@
+package com.topic1;
+
+import java.util.HashSet;
+import java.util.Iterator;
+
+public class MailList implements Subject {
+
+	HashSet<Observer> observers;
+	String change;
```

Comment:
[High] You're not using it: Neither initialize it nor update it. Also it doesn't have the access modifier (private, public, protected)